CGPs/cgp-0093.md

```diff
@@ -0,0 +1,314 @@
+- **Venue**: 25,000 cEUR
+- **Catering**: 15,000 cEUR
+- **Production**: 7,500 cEUR
+- M**erchandise**: 2,500 cEUR
+
+
+## Known Risks
+Informed by our knowledge in the blockchain and climate space, we recognize the market risk posed by the current macro economic context, as well as the regulatory uncertainty chilling much of the crypto space. We’ve mitigated this risk by seeking a smaller initial pool of funding while we prove our product hypothesis and growth model before developing production-grade software.
+
+We also recognize product development risk as a core facet of any new technology.
+
+These risks are mitigated through world-class research and design to ensure any future engineering efforts result in products people want to use and behaviors that will ultimately result in impact and ecosystem growth for Celo blockchain.
+
+## Core team
+We’re currently operating a small, lean core team assisted by a suite of AI co-pilots. We are actively working with leading AI experts at [AI.PT](http://AI.PT) to improve our workflows and operational efficiencies across every aspect of our organization.
+
+Members currently include:
+- John Ellison 🇺🇸 — serial entrepreneur, behavior designer, podcast host (ex-Toucan Protocol, OpenGov)
+- Emilie Viallon 🇫🇷 — HR executive, kundalini yoga instructor (ex-Checkout.com)
+- Rica Amaral  🇧🇷 — positive blockchain, web3, and impact pioneer, former Celo employee
+- Monty Merlin Bryant 🇬🇧 — ReFi advocate, Master in impact and innovation from Bristol University
+- Anna Kaic 🇭🇷 — multi-lingual marketer and growth lead (ex-ReFi Spring, Urbanika)
+
+Additional members:
+- Community Manager - we will be looking for a full-time community manager as a part of this grant.
+
+We’ll be compensating the core team with a temporarily reduced mean salary of 3,333 cEUR per month, with an expectation to increase back-to-market rates as we gain further funding. 
+
+### Costs
+- 6 core team members at an average of 3,333 cEUR per month = 60,000 EUR (approx.132,000 CELO)
+
+## Advisors
+We have been grateful to receive support from the following strategic advisors:
+- Phil Fogel — Flowcarbon
+- Patrick Rawson — Curve Labs
+- Simar Mangat — Magic Ventures
+- Ian Poirier — Toucan Protocol
+
+## Road to sustainability
+We intend to turn the ReFi DAO into a self-sustaining organization by the end of 2025 in two main ways:
+1. Membership Fee 
+2. Protocol Fee
+3. Event and content sponsorship 
+
+The first two long-term value capture mechanisms need to be designed and tested through product discovery and implementation. We intend to raise further funds from value-aligned capital allocators as the market sentiment continues to improve alongside our ongoing Gitcoin fundraising efforts.
+
+We’ve already proven we can gather event sponsorship revenue at ReFi Lisboa with a self-sustaining event business in just 6 months.
+
+## Multisig
+* **Multisign Address:** XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
We will need a multisig set up before we can merge this. The Gnosis safe integration is what most proposers use.